code/modules/mob/living/carbon/carbon_life.dm

```diff
@@ -293,10 +293,15 @@
 /mob/living/carbon/update_damage_hud()
 	if(!client)
 		return
+	var/shock_reduction = 0
+	if(reagents)
+		for(var/datum/reagent/R in reagents.reagent_list)
+			if(R.shock_reduction)
+				shock_reduction += R.shock_reduction
```

Comment:
Can you make this into its own proc? Since this code is used twice now.

Reply:
Done!